client/Lykke.Service.Balances.Client/IBalancesClient.cs

```diff
@@ -13,5 +13,6 @@ public interface IBalancesClient
         Task<WalletCredentialsModel> GetWalletCredential(string clientId);
         Task<WalletCredentialsHistoryModel> GetWalletCredentialHistory(string clientId);
         Task<IEnumerable<ClientBalanceResponseModel>> GetTotalBalances();
+        Task<ClientBalanceResponseModel> GetTotalBalance(string assetId);
```

Comment:
GetTotalBalanceAsync?

Reply:
Decided to name a method without ...Async - since all other methods are also w/o it